Classes/Handler/Exception/MissingArgumentsException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pixelant\Interest\Handler\Exception;
+
+class MissingArgumentsException extends AbstractRequestHandlerException
```

Comment:
I think this should be singular.

```suggestion
class MissingArgumentException extends AbstractRequestHandlerException
```